src/styles.css

```
@@ -121,6 +122,9 @@
   }
   body {
     @apply bg-background text-foreground;
+    background:
+      radial-gradient(circle at 20% -10%, rgb(42 116 255 / 0.16), transparent 28rem),
+      radial-gradient(circle at 90% 5%, rgb(143 77 255 / 0.14), transparent 24rem), #101014;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)